packages/babel-parser/test/fixtures/estree/private-in/basic/input.js

```diff
@@ -0,0 +1,6 @@
+class A {
+  #foo = "bar";
+  static isA(obj) {
+    return #foo in obj;
```

Comment:
Since the estree private brand checks proposal is also moved to stage 3. I just add a test case for private-in.